metaflow/metaflow_config.py

```diff
@@ -45,6 +45,12 @@ def from_conf(name, default=None):
 DEFAULT_DATASTORE = from_conf('METAFLOW_DEFAULT_DATASTORE', 'local')
 DEFAULT_METADATA = from_conf('METAFLOW_DEFAULT_METADATA', 'local')
 
+##
+# KFP core information
```

Comment:
core information? What is core information?

Reply:
Also why use a KFP_SDK_USERID when Metaflow has a username or METAFLOW_USER?  Can you work on mapping METAFLOW_USER to the KFP user?

Reply:
Resolved for now.